src/main/resources/application.properties

```diff
@@ -86,6 +86,13 @@ quarkus.opentelemetry.tracer.exporter.otlp.enabled=true
 #  Override Log Level with Environment Variable QUARKUS_LOG_LEVEL
 quarkus.log.level=INFO
 
+# Set maximum upload size to 100mb
+quarkus.http.limits.max-body-size=100M
+
+# Don't scan for code-first endpoints to merge with static API contract
+# suppress inspection "UnusedProperty"
+mp.openapi.scan.disable=true
```

Comment:
I am sure this is needed, but what exactly is this doing?

Reply:
This is telling Quarkus's SmallRye OpenAPI extension to not scan for REST endpoints in the application for purposes of building an OpenAPI Specification document. (The SmallRye OpenAPI library uses MicroProfile property names; hence the "mp.openapi" prefix.)

In other words, this prevents Quarkus/SmallRye from adding or changing the definition of endpoints listed in the OpenAPI Specification at runtime. As such, only the OpenAPI document provided at classpath:/META-INF/openapi.yml will be made available at the path /q/openapi

I'll expand the comment about this in the application.properties file for easy reference in the future.